prow/oss/config.yaml

```diff
@@ -94,6 +94,7 @@ tide:
     labels:
     - lgtm
     - approved
+    - ci-passed
```

Comment:
@VaniHaripriya this will probably change the config for [all projects](https://github.com/GoogleCloudPlatform/oss-test-infra/tree/master/prow/prowjobs). We need to figure out a way to set this config only for [KFP](https://github.com/GoogleCloudPlatform/oss-test-infra/tree/master/prow/prowjobs/kubeflow/pipelines).